src/libsync/progressdispatcher.cpp

```diff
@@ -229,7 +245,7 @@ void ProgressInfo::setProgressComplete(const SyncFileItem &item)
     _currentItems.remove(item._file);
```

Comment:
needs to move below the `if` or `_totalSizeOfCompletedJobs` won't update correctly